src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="compress_values">
```

Comment:
I'd mark this as `translatable=false` just to be sure. Also you forgot a newline at the file end ;)